build/build-info

```diff
@@ -32,6 +32,7 @@ echo_build_properties() {
   echo branch=$(cd "$git_path" && git rev-parse --abbrev-ref HEAD)
   echo date=$(date -u +%Y-%m-%dT%H:%M:%SZ)
   echo url=$(cd "$git_path" && git config --get remote.origin.url)
+  echo gpu_architectures=$(cd "$git_path" && find . -name libcudfjni.a -o -name libcudf.a | xargs -n 1 bash -c 'cuobjdump $1 || exit 0' _ | grep 'arch = ' | awk -F_ '{print $2}' | sort -n -u)
```

Comment:
I am not in favor of the space-separated arch list. Java properties file format allows for the space being the separator between the key and the value portion of the line
```Java
jshell> var props = new java.util.Properties()
props ==> {}

jshell> props.load(new java.io.StringReader("gpu_architectures 70 75 80 86 90"))

jshell> props
props ==> {gpu_architectures=70 75 80 86 90}
```

I prefer a semicolon-separated value format without any white space in the value  `70;75;80;86;90` similar to the first example https://cmake.org/cmake/help/latest/prop_tgt/CUDA_ARCHITECTURES.html#prop_tgt:CUDA_ARCHITECTURES and this is what we typically use for `-DGPU_ARCHS` passed to Maven

Reply:
That makes sense. This would keep it consistent with current usage.